terraform-modules/aws/airflow/default_iam_policy.json

```diff
@@ -84,7 +85,7 @@
                 "kms:GenerateDataKey*",
                 "kms:Encrypt"
             ],
-            "Resource": "arn:aws:kms:${aws_region}:${aws_account_id}:key/*",
+            "Resource": "*",
```

Comment:
I was working hard to find out why this only works with asterisk, and in the end chat gtp helped me with the answer since there is little documentation.

**chat gpt**
if you are using the default aws/airflow (which is our case in airflow with default policy) KMS key, you do not need to include a specific policy for KMS in your IAM role. The necessary permissions to use this key are already granted by the service to your Amazon MWAA environment.
Since you are using the default aws/airflow KMS key, you cannot specify its ARN directly in the policy. You can set the "Resource" field to "*" to allow access to all KMS keys in your account

Reply:
this solved the issued in airflow
![image](https://user-images.githubusercontent.com/19688747/234432203-ca6dbdb6-553a-437c-996f-d37b034cc34a.png)